src/util/getOwnersOfPackages.ts

```diff
@@ -8,16 +8,23 @@ import { GetFileContent } from "../queries/file-query";
 
 import { GetFileContent as GetFileContentResult } from "../schema/GetFileContent";
 
-export async function getOwnersOfPackages(packages: readonly string[]) {
-  const allOwners = new Set<string>();
```

Comment:
Being a Set (i.e., needing custom serialization/deserialization) was a lot more trouble than it was worth. Insertion is now O(n^2)-ish but seems unlikely to matter (and could be optimized)